lib/deep_nest.rb

```diff
@@ -1,136 +1,24 @@
 # frozen_string_literal: true
 
-require_relative "deep_nest/version"
+require_relative 'deep_nest/helpers'
+require_relative 'deep_nest/version'
 
 ##
 # Primary namespace for the deep_nest gem.
 module DeepNest
+  ##
+  # Default error class for DeepNest.
   class Error < StandardError; end
-  class << self
```

Comment:
We don't want to remove this work - it should stay as-is. We want users to be able to use the standalone methods as easily as possible. So we don't want them to have to call `DeepNest::Methods.deep_merge`, just `DeepNest.deep_merge`. 

This task is about adding functionality and making it more convenient, without taking away anything we've already built.